README.md

```diff
@@ -1,23 +1,27 @@
 ## Installation
 
-Once QIIME2 is [installed](https://docs.qiime2.org/2017.5/install/native/), it should be possible to install `q2-fragment-insertion` with:
+Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), it should be possible to install `q2-fragment-insertion` with:
 
-    source activate qiime2-2017.5
-    git clone https://github.com/wasade/q2-fragment-insertion.git
+    source activate qiime2-2017.10
+    git clone https://github.com/biocore/q2-fragment-insertion.git
     cd q2-fragment-insertion
     python setup.py install
     qiime dev refresh-cache
-    
+
 You will need Java to run SEPP, and you may need to install it. How you do it is dependent on your operating system. If you're on OSX, you likely need to grab a legacy version of Java. Information can be found [here](https://support.apple.com/kb/dl1572?locale=en_US).
```

Comment:
conda install java?

Reply:
I remember having issues installing java via conda on the Mac Tower.